benchmarks/synchrobench/src/main/java/com/yahoo/oak/synchrobench/data/eventcache/ValueGen.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright 2020, Verizon Media.
+ * Licensed under the terms of the Apache 2.0 license.
+ * Please see LICENSE file in the project root for terms.
+ */
+
+package com.yahoo.oak.synchrobench.data.eventcache;
+
+import com.yahoo.oak.OakBuffer;
+import com.yahoo.oak.OakScopedReadBuffer;
+import com.yahoo.oak.OakScopedWriteBuffer;
+import com.yahoo.oak.UnsafeUtils;
+import com.yahoo.oak.synchrobench.contention.abstractions.BenchValue;
+import com.yahoo.oak.synchrobench.contention.abstractions.ValueGenerator;
+import net.openhft.chronicle.bytes.Bytes;
+import net.spy.memcached.CachedData;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.openjdk.jmh.infra.Blackhole;
+import sun.misc.Unsafe;
+
+import java.util.Random;
+
+public class ValueGen implements ValueGenerator {
```

Comment:
Why both `ValueGen` and `ValueGenerator`? Why so many value offsets? Seems like comments per classes are very missed

Reply:
`ValueGenerator` is the interface. I can't name the class and interface with the same name.